controllers/openstackdataplanenode_controller.go

```diff
@@ -138,7 +138,8 @@ func (r *OpenStackDataPlaneNodeReconciler) Reconcile(ctx context.Context, req ct
 	// persist any changes.
 	defer func() {
 		// update the overall status condition if service is ready
-		if instance.IsReady() {
+		if instance.IsReady() || instanceRole.IsReady() {
+			instance.Status.Deployed = true
```

Comment:
there's nothing to trigger a node reconcile so that this would get run

Reply:
Yea, i think we need that loop back. This worked for me:

```
Mon Mar 27 01:55:14 (jslagle@teletran-1) [~/code/github/openstack-k8s-operators/dataplane-operator] (nodestatus)$ git diff
diff --git a/controllers/openstackdataplanerole_controller.go b/controllers/openstackdataplanerole_controller.go
index aa35f12..1af8bbf 100644
--- a/controllers/openstackdataplanerole_controller.go
+++ b/controllers/openstackdataplanerole_controller.go
@@ -282,6 +282,15 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
                r.Log.Info("Set ReadyCondition true")
                instance.Status.Conditions.Set(condition.TrueCondition(condition.ReadyCondition, dataplanev1beta1.DataPlaneRoleReadyMessage))
 
+               for _, node := range nodes.Items {
+                       _, err := controllerutil.CreateOrPatch(ctx, helper.GetClient(), &node, func() error {
+                               node.Status.Deployed = true
+                               return nil
+                       })
+                       if err != nil {
+                               return ctrl.Result{}, err
+                       }
+               }
                // Explicitly set instance.Spec.Deploy = false
                // We don't want another deploy triggered by any reconcile request, it should
                // only be triggered when the user (or another controller) specifically
```

Reply:
done!